radix-engine-interface/src/blueprints/resource/access_rules.rs

```diff
@@ -9,6 +9,8 @@ use sbor::rust::collections::BTreeMap;
 use sbor::rust::str;
 use sbor::rust::string::String;
 use sbor::rust::string::ToString;
+use utils::rust::prelude::index_map;
+use utils::rust::prelude::IndexMap;
```

Comment:
You could/should just replace all of these with `import sbor::rust::prelude::*` or `import utils::rust::prelude::*`